test/deim.jl

```diff
@@ -0,0 +1,61 @@
+using Test, ModelOrderReduction
+using ModelingToolkit, MethodOfLines, DifferentialEquations
+
+# construct an ModelingToolkit.ODESystem with non-empty field substitutions
+@variables x t v(..) w(..)
+Dx = Differential(x)
+Dxx = Dx^2
+Dt = Differential(t)
+const L = 1.0
+const ε = 0.015
+const b = 0.5
+const γ = 2.0
+const c = 0.05
+f(v) = v * (v - 0.1) * (1.0 - v)
+i₀(t) = 50000.0t^3 * exp(-15.0t)
+eqs = [ε * Dt(v(x, t)) ~ ε^2 * Dxx(v(x, t)) + f(v(x, t)) - w(x, t) + c,
+    Dt(w(x, t)) ~ b * v(x, t) - γ * w(x, t) + c]
+bcs = [v(x, 0.0) ~ 0.0,
+    w(x, 0) ~ 0.0,
+    Dx(v(0, t)) ~ -i₀(t),
+    Dx(v(L, t)) ~ 0.0]
+domains = [x ∈ (0.0, L),
+    t ∈ (0.0, 14.0)]
+ivs = [x, t]
+dvs = [v(x, t), w(x, t)]
+pde_sys = PDESystem(eqs, bcs, domains, ivs, dvs; name = :FN)
+
+N = 5 # (minimum number of) equidistant discretization intervals
```

Comment:
Here, I aimed to focus on testing the fields `defaults`, `observed`, etc of `ODESystem` are set correctly such that users can recover eliminated variables without any trouble after solving the reduced system. In the meanwhile, I would like to avoid causing burden to the test CI. And normally benchmark is not put in unittests. Therefore, I set the number of discretization intervals `N` to the minimum number.

Reply:
yeah, you are absolutely right. For some reason I confused it with the documented example. Not sure why. This is perfectly fine for CI.